src/react-apps/applications/service-development/src/config/sharedConfig.ts

```diff
@@ -0,0 +1,5 @@
+export const urls = {
+  apps: {
+    test: 'altinn.studio',
```

Comment:
isnt the test cluster supposed to have altinn.cloud as ending? 'altinn.studio' will be wrong here, I saw that you wrote that it was temporary, but in this case altinn.cloud would be more correct

Reply:
True.

Currently our only option is to create certificates for altinn.studio. When we have the option to create certificates for altinn.cloud, this have to change, and all parts will be changed.

Reply:
Added apps + altinn objects with cloud and studio.